app/src/main/java/com/greenfox/peridot/peridot_coz_android/api/MockService.java

```diff
@@ -36,6 +37,25 @@
     public MockService() {}
 
     @Override
+    public Call<UserResponse> searchUser() {
+        return null;
+    }
+
+    @Override
+    public Call<UsersResponse> getUsers() {
+        return new MockCall<UsersResponse>() {
+            @Override
+            public void enqueue(Callback<UsersResponse> callback) {
+                users.add(new User("Balint"));
+                users.add(new User("Bea"));
+                users.add(new User("Szilvi"));
```

Comment:
@Phaedra144 move these to the constructor so it wont add them everytime this gets called